Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -2186,6 +2190,7 @@ RefPtr<API::Navigation> WebPageProxy::loadFile(const String& fileURLString, cons
     loadParameters.userData = UserData(legacyMainFrameProcess().transformObjectsToHandles(userData).get());
     loadParameters.publicSuffix = WebCore::PublicSuffixStore::singleton().publicSuffix(loadParameters.request.url());
     loadParameters.isRequestFromClientOrUserInput = isAppInitiated;
+    loadParameters.isHandledByAboutSchemeHandler = AboutSchemeHandler::singleton().canHandleURL(loadParameters.request.url());
```

Comment:
We're in `loadFile()`, why is this needed? Surely a file URL cannot be handled by the about scheme handler?